internal/ingress/controller/kong_test.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2018 Kong Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import (
+	"testing"
+
+	kongadminv1 "github.com/kong/kubernetes-ingress-controller/internal/apis/admin/v1"
+)
+
+func TestPluginDeepEqual(t *testing.T) {
+	var equal bool
+	equal = pluginDeepEqual(map[string]interface{}{}, &kongadminv1.Plugin{Config: map[string]interface{}{}})
+	if !equal {
+		t.Errorf("Comparation empty maps  failed on pluginDeepEqual")
```

Comment:
s/Comparation/Comparing

There is an extra space between maps and failed.

Also, "on pluginDeepEqual" is not necessary here since Go's testing framework will communicate that `TestPluginDeepEqual` failed. Can we remove that and make the error line as:
> Comparing empty configs failed

Could you change it for other tests in this file? Thanks!

Reply:
Added!